packages/editor/src/Editor.tsx

```diff
@@ -25,12 +26,22 @@ const StyledEditor = styled(Editor)`
 `;
 
 export class CivilEditor extends React.Component<EditorProps, EditorState> {
+<<<<<<< Updated upstream
```

Comment:
is this right? looks like it is a merge conflict

Reply:
I thought i got rid of those :(